frontend/src/app/components/wp-activity/revision/revision-activity.component.html

```diff
@@ -10,13 +10,12 @@
 
   <user-avatar *ngIf="activity.author && userName"
                [attr.data-user-name]="userName"
-               [attr.data-user-avatar-src]="userAvatar"
+               [attr.data-user-id]="userId"
                data-class-list="avatar">
   </user-avatar>
 
   <user-avatar *ngIf="!activity.author && userName"
                [attr.data-user-name]="userName"
-               [attr.data-use-fallback]="true"
```

Comment:
Is there a reason you removed this? We could save the request for an invalid url, as we know for sure that there is no avatar available here.

Reply:
Good point, will re-add it